### PR TITLE
address VS-152, remove extra headers from extract

### PR DESCRIPTION
**Pull request by kcibul**

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java [hunk 1]
```diff
@@ -1,9 +1,7 @@
 package org.broadinstitute.hellbender.tools.gvs.extract;
 
-import htsjdk.variant.vcf.VCFFormatHeaderLine;
-import htsjdk.variant.vcf.VCFHeader;
-import htsjdk.variant.vcf.VCFHeaderLine;
-import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.vcf.*;
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:4]: was this intentional? Are we changing styles?

Comment (kcibul) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:4]: intellij autoformatting -- I didn't even know we had an official style :D but happy to change it back

File: scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def get_all_sample_ids(fq_destination_table_samples):
 
 def create_extract_samples_table(fq_destination_table_samples, fq_sample_name_table, fq_sample_mapping_table):
   sql = f"CREATE OR REPLACE TABLE `{fq_destination_table_samples}` AS (" \
-        f"SELECT m.sample_id, m.sample_name FROM `{fq_sample_name_table}` s JOIN `{fq_sample_mapping_table}` m ON (s.sample_name = m.sample_name) " \
+        f"SELECT m.sample_id, m.sample_name, s.is_loaded FROM `{fq_sample_name_table}` s JOIN `{fq_sample_mapping_table}` m ON (s.sample_name = m.sample_name) " \
```
Comment (ahaessly) [inline scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:124]: this should be `m.is_loaded` the s table might be the temp table with only sample names

**Review by ahaessly — APPROVED** [review]
only one change needed

File: scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py (hunk 2 above)
Comment (kcibul) [reply scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:124]: ah great catch

**Pull request merged**